redhat-img/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM registry.access.redhat.com/rhel7
```

Comment:
I think it's better to specify the version of the base OS image. Otherwise, we don't even know which version will be used when running docker build.

Reply:
you mean add a specific tag?  the latest version is `7.4-129`
https://access.redhat.com/containers/?tab=overview#/registry.access.redhat.com/rhel7


However, the [examples from redhat](https://github.com/RHsyseng/container-rhel-examples/blob/master/starter/Dockerfile) do not specify a tag.

Reply:
So we change it to this, and update it regularly.
```yaml
FROM registry.access.redhat.com/rhel7:7.4-129
```